CMP1903_A1_2324/Die.cs

```diff
@@ -8,16 +8,19 @@ namespace CMP1903_A1_2324
 {
     internal class Die
     {
-        /*
-         * The Die class should contain one property to hold the current die value,
-         * and one method that rolls the die, returns and integer and takes no parameters.
-         */
-
-        //Property
-
-
-        //Method
+        private Random random;
+        public int dieRolled {get; private set;}
+        public Die()
+        {
+            random = new Random();
```

Comment:
The code uses the random object in the constructor to avoid null reference exceptions when rolling the die.